src/python_testing/matter_testing_infrastructure/chip/testing/choice_conformance.py

```diff
@@ -53,7 +54,8 @@ def evaluate_attribute_choice_conformance(endpoint_id: int, cluster_id: int, xml
     for attribute_id in all_attributes:
         conformance_decision_with_choice = xml_clusters[cluster_id].attributes[attribute_id].conformance(
             feature_map, attribute_list, all_command_list)
-        _add_to_counts_if_required(conformance_decision_with_choice, attribute_id in attribute_list, counts)
+        if isinstance(conformance_decision_with_choice, ConformanceDecisionWithChoice):
```

Comment:
All the conformance calls should return ConformanceDecisionWithChoice - is there a way to fix the type markings so mypy can deduce this? Alternately, if there's anything not returning that, we should know about it because that's not correct.